core/config/locales/en.yml

```diff
@@ -720,7 +726,7 @@ en:
     average_order_value: Average Order Value
     avs_response: AVS Response
     back: Back
-    back_end: Backend
+    back_end: Back End
```

Comment:
This translation is not used in this PR, and I'm not sure whether `Back End` is actually more correct than `Backend`.